epersongroups.md

```diff
@@ -0,0 +1,310 @@
+          "name": "atmirenv@gmail.com",
+          "handle": null,
+          "metadata": [
+            {
+              "key": "eperson.firstname",
+              "value": "Atmire",
+              "language": null
+            },
+            {
+              "key": "eperson.lastname",
+              "value": "NV",
+              "language": null
+            },
+            {
+              "key": "eperson.language",
+              "value": "en",
+              "language": null
+            }
+          ],
+          "netid": null,
+          "lastActive": "2018-12-19T11:06:43.404+0000",
+          "canLogIn": true,
+          "email": "atmirenv@gmail.com",
+          "requireCertificate": false,
+          "selfRegistered": false,
+          "groups": null,
+          "type": "eperson",
+          "_links": {
+            "self": {
+              "href": "https://dspace7.4science.it/dspace-spring-rest/api/eperson/epersons/a6086b34-3918-45b7-8ddd-9329a702a26a"
+            }
+          }
+        }
+      ]
+    },
+    "_links": {
+      "self": {
+        "href": "https://dspace7.4science.it/dspace-spring-rest/api/eperson/groups/3b1de75d-5bf9-4ca3-bf4d-42bc8abd0d42/epersons"
+      }
+    },
+    "page": {
+      "number": 0,
+      "size": 10,
+      "totalPages": 1,
+      "totalElements": 10
+    }
+}
+```
+
+### Add an EPerson to a parent Group
+
+**POST /api/eperson/groups/<:groupuuid>/epersons**
+
+To add an eperson to a parent group, perform a POST to the Epeople of a Group endpoint when logged in as admin.
+
+The actual eperson is part of the body using the uri-list
+
+Example:
+```bash
+curl -i -X POST "https://dspace7.4science.it/dspace-spring-rest/api/eperson/groups/617cf46b-535c-42d5-9d22-327ce2eff6dc/epersons"
+  -H "Content-Type:text/uri-list"
+  -d "https://dspace7.4science.it/dspace-spring-rest/api/eperson/epersons/a6086b34-3918-45b7-8ddd-9329a702a26a"
+```
+
+The eperson is mandatory
+
+Return codes:
+* 204: if the update succeeded 
+* 401 Forbidden - if you are not authenticated
+* 403 Unauthorized - if you are not logged in with sufficient permissions 
+* 422: if the specified eperson is not found
+
+### Remove an EPerson from a parent Group
+
+**DELETE /api/eperson/groups/<:groupuuid>/epersons**
+
+To remove an eperson from a parent group, perform a DELETE to the Epeople of a Group endpoint when logged in as admin.
+
+The actual eperson is part of the body using the uri-list
+
+Example:
+```bash
+curl -i -X DELETE "https://dspace7.4science.it/dspace-spring-rest/api/eperson/groups/617cf46b-535c-42d5-9d22-327ce2eff6dc/epersons"
+  -H "Content-Type:text/uri-list"
+  -d "https://dspace7.4science.it/dspace-spring-rest/api/eperson/epersons/a6086b34-3918-45b7-8ddd-9329a702a26a"
+```
+
+The eperson is mandatory
+
+Return codes:
+* 204: if the update succeeded 
+* 401 Forbidden - if you are not authenticated
+* 403 Unauthorized - if you are not logged in with sufficient permissions 
+* 422: if the specified eperson is not found
```

Comment:
this is ambiguous. I guess that we mean that the specified eperson is not found in the system, it doesn't exist at all. It can be misunderstood as the specified eperson is not found in the group. In this later case, I guess that 204 should be returned with a no-op

Reply:
This has been been updated